test/e2e/storage/testsuites/provisioning.go

```diff
@@ -566,7 +566,7 @@ func (p *provisioningTestSuite) DefineTests(driver storageframework.TestDriver,
 		}
 		// Cloning fails if the source disk is still in the process of detaching, so we wait for the VolumeAttachment to be removed before cloning.
 		volumeAttachment := e2evolume.GetVolumeAttachmentName(ctx, f.ClientSet, testConfig, l.testCase.Provisioner, dataSourceRef.Name, l.sourcePVC.Namespace)
-		e2evolume.WaitForVolumeAttachmentTerminated(ctx, volumeAttachment, f.ClientSet, f.Timeouts.DataSourceProvision)
+		framework.ExpectNoError(e2evolume.WaitForVolumeAttachmentTerminated(ctx, volumeAttachment, f.ClientSet, f.Timeouts.DataSourceProvision))
```

Comment:
The error returned by `WaitForVolumeAttachmentTerminated` is informative enough to not need further annotations of the error check.